server/fleet/vulnerabilities.go

```diff
@@ -136,19 +136,20 @@ type VulnerabilityWithMetadata struct {
 }
 
 type VulnListOptions struct {
-	ListOptions
+	// ListOptions cannot be embedded in order to unmarshall with validation.
```

Comment:
Great catch!

I will be adding some code (on another PR) to prevent this from happening again.

Reply:
Here it is: https://github.com/fleetdm/fleet/pull/18172